### DISCUSSION TITLE
multiple major versions in a Regex

### DISCUSSION THREAD

**Discussion by der-ali** [renovatebot/renovate#27358] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

37.0.0

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### What is your question?

I created this packagerule
```
"packageRules": [
    {
      "matchManagers": ["terraform"],
      "matchDepTypes": ["module"],
      "versioning": "regex:^((?<major>\\d+)\\.(?<minor>\\d+)\\.(?<patch>\\d+))\\-((?<first>\\d+)\\.(?<second>\\d+)\\.(?<third>\\d+))$"
    }
  ]
```
in-order to configure Renovatebot to detect and update terraform modules with tags similar to `1.1.0-0.0.1`.
Renovate bot is able to detect the dependency, but probably treating the second versioning group aka. 0.0.1 as compatibility.
Any help please?

**Comment by viceice** [comment]
you can currently only capture 5 numbers and everything below patch is a patch update

https://docs.renovatebot.com/modules/versioning/#regular-expression-versioning
